src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const bodyDoc = document.querySelector('body');
+const mateLogo = document.querySelector('.logo');
+
+const promise1 = () => {
+  return new Promise((resolve, reject) => {
+    mateLogo.addEventListener('click', () => resolve('Promise was resolved!'));
```

Comment:
The 'click' event listener added to `mateLogo` in `promise1` will never be removed. This could lead to memory leaks if `promise1` is called multiple times. Consider removing the event listener after the promise is resolved or rejected.